Updates on Restore with MCDB (#2018)
MicrosoftDocs/OfficeDocs-Exchange: Exchange/ExchangeServer2013/restore-data-using-a-recovery-database-exchange-2013-help.md

- Some backup applications have the ability to restore Exchange data directly to a recovery database. Windows Server Backup can restore only file-level backups to a recovery database. It cannot be used to restore application-level backups to a recovery database.

- The database and log files containing the recovered data must be restored or copied into the RDB folder structure. Do not rename the .edb files.

> - The database must be in a clean shutdown state. If the previously restored databases is in a dirty shutdown state. You must use **Eseutil /R** to put restored databases into a clean shutdown state. If you are running Exchange 2019 and utilizing a MCDB you must use **Eseutil /R /i**

Reviewer: ```suggestion
- The database must be in a clean shutdown state. If the previously restored databases are in a dirty shutdown state, you must use **Eseutil /R** to put restored databases to a clean shutdown state. If you are running Exchange 2019 and utilizing an MCDB, you must use **Eseutil /R /i**.
```
